### PR TITLE
In-progress improvements to documentation

### PR DESCRIPTION
**Pull request by misterbisson**
1. Run network setup in a single line on Macs, though I'm not sure how to do the same on Windows.

### CODE REVIEW

**Comment by lloydde** [comment]
@misterbisson if this work is in-progress, maybe wait until complete before pull request? (and tested it on Windows, "tools" folder would not exist ;-) )

File: README.md [hunk 1]
```diff
@@ -108,22 +108,26 @@ See [CoaL Setup](./docs/developer-guide/coal-setup.md) for a thorough walkthroug
...
          - Windows:
 
-            ```
-            tools\coal-windows-vmware-setup.bat
-            ```
+            1. Download [the script](https://github.com/joyent/sdc/blob/master/tools/coal-windows-vmware-setup.bat) and save it to a local file.
+            1. Execute the script:
+
+                ```
+                tools\coal-windows-vmware-setup.bat
```
Comment (misterbisson) [inline README.md]: This is copied verbatim from the original, just tabbed in a bit.

**Comment by misterbisson** [comment]
@lloydde your points are fair, though I often like to open PRs for in-progress changes to get early feedback that might change my next steps. In this case it makes it easier to ask for feedback on how to do on Windows the change I'm proposing for Macs.

File: README.md (hunk 1 above)
Comment (misterbisson) [inline README.md]: Suggestion via https://twitter.com/trevorsuarez/status/552539937590231040 :

> I know you can do it with Powershell. A good example is on the Chocolatey homepage: https://chocolatey.org/ 

That would look something like this, I think:

```
PS:\>iex ((new-object net.webclient).DownloadString('https://github.com/joyent/sdc/blob/master/tools/coal-windows-vmware-setup.bat'))
```

**Comment by lloydde** [comment]
@misterbisson that isn't the convention I'm most familiar with in system software pull requests. There are plenty of people to reach out directly to or through channels like jabber or IRC. A pull request pushes notifications to everyone subscribed.

**Comment by lloydde** [comment]
Closing pull request for now.

**Pull request closed without merging**